cmd/query/app/apiv3/grpc_handler_test.go

```diff
@@ -164,6 +167,40 @@ func TestFindTraces(t *testing.T) {
 	require.EqualValues(t, 1, td.SpanCount())
 }
 
+func TestFindTracesSendError(t *testing.T) {
+	reader := new(spanstoremocks.Reader)
+	reader.On("FindTraces", mock.Anything, mock.AnythingOfType("*spanstore.TraceQueryParameters")).Return(
+		[]*model.Trace{
+			{
+				Spans: []*model.Span{
+					{
+						OperationName: "name",
+					},
+				},
+			},
+		}, nil).Once()
+	h := &Handler{
+		QueryService: querysvc.NewQueryService(
+			factoryadapter.NewTraceReader(reader),
+			new(dependencyStoreMocks.Reader),
+			querysvc.QueryServiceOptions{},
+		),
+	}
+	err := h.internalFindTraces(context.Background(),
+		&api_v3.FindTracesRequest{
+			Query: &api_v3.TraceQueryParameters{
+				StartTimeMin: time.Now().Add(-2 * time.Hour),
+				StartTimeMax: time.Now(),
+			},
+		},
+		/* streamSend= */ func(*api_v3.TracesData) error {
+			return errors.New("send_error")
```

Comment:
nit: you can define this error outside and then you won't need to do a string comparison - you can just do `require.ErrorIs`

Reply:
I am not sure it will work (because of grpc error wrapping). It's not a guarantee of the gRPC API (cross-process in real life) to propagate error in such a way that `ErrorIs` would work.